lib/core/test/unit/Cardano/WalletSpec.hs

```diff
@@ -151,6 +166,8 @@ spec = do
             (property walletUpdatePassphraseDate)
         it "Root key is re-encrypted with new passphrase"
             (withMaxSuccess 10 $ property walletKeyIsReencrypted)
+        it "Wallet can list transactions"
+            (withMaxSuccess 10 $ property walletListTransactionsSorted)
```

Comment:
I think there's no reason to limit this one to `10` successes, the one above is because it does heavy cryptographic work.. This one doesn't.